### PR TITLE
chore: github templates for issues

### PR DESCRIPTION
**Pull request by oap75**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+
+name: 🐛 Bug Report
+about: Template for reporting bugs
+title: '[BUG] <title>'
```
Comment (endersonmaia) [inline .github/ISSUE_TEMPLATE/bug-report.md:5]:
```suggestion
title: '<title>'
```

We already have the `bug` label.

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+name: ✨ Feature request
+about: Template for requesting new features
+title: "[FEATURE] <title>"
```
Comment (endersonmaia) [inline .github/ISSUE_TEMPLATE/feature-request.md:4]:
```suggestion
title: "<title>"
```

We already have a `enhancement` label.

**Review by endersonmaia — CHANGES_REQUESTED** [review]
We can target the `main` brach here, since we don't have changes to any charts.

We could merge directly there to use it already.

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 1 above)
Comment (oap75) [reply .github/ISSUE_TEMPLATE/bug-report.md]: Fixed !

File: .github/ISSUE_TEMPLATE/feature-request.md (hunk 2 above)
Comment (oap75) [reply .github/ISSUE_TEMPLATE/feature-request.md]: Fixed !

**Review by endersonmaia — APPROVED** [review]

**Pull request merged**